string coord
time timestamp
PandarPacket[] packets
